packages/table/src/regions.ts

```diff
@@ -169,6 +169,16 @@ export class Regions {
     }
 
     /**
+     * Returns a deep copy of the provided region.
+     */
+    public static copy(region: IRegion): IRegion {
```

Comment:
Shallower copies from spreads: `{ ...region }` were causing `shouldComponentUpdate` checks to pass erroneously early in this work. Introduced this function as a hammer; may not need this though. LMK what you think.